src/main/java/com/xceptance/xlt/api/engine/Session.java

```diff
@@ -115,6 +115,13 @@ public static final void logEvent(final String eventName, final String eventMess
      * @return the data manager
      */
     public abstract DataManager getDataManager();
+    
+    /**
+     * Logs a line of custom data in the datalogger of the given scope.
+     * (Shortcut method for getDataManager().dataLogger(scope).log(lineOfData))
+     *
+     */
+    public abstract void logData(String scope, String lineOfData);
```

Comment:
PO wanted `Session.logData(scope, data)` as shortcut, not `Session.getCurrent().logData(scope, data)`. Please rework this accordingly. See `Session.logEvent()` for inspiration.

In this course, please add parameter descriptions to the javadoc.